modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -29,7 +29,7 @@ To dynamically create a volume snapshot:
 .volumesnapshotclass.yaml
 [source,yaml]
 ----
-apiVersion: snapshot.storage.k8s.io
+apiVersion: snapshot.storage.k8s.io/v1beta1
 kind: VolumeSnapshotClass <1>
```

Comment:
@bobfuru
Is this annotation put in the right place? Not sure which attributes mentioned here 
<1> Allows you to specify different attributes belonging to a volume snapshot.

Reply:
Not sure either, it was in there from TP. I've removed the annotation.